.vib/influxdb/cypress/cypress/integration/influxdb_spec.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="cypress" />
+import { random, selectOrg } from '../support/utils';
+
+it('can create a new bucket', () => {
+  cy.login();
+  selectOrg();
+  cy.visitInOrg('/load-data/buckets');
+  cy.get('[data-testid="Create Bucket"]').click();
+  cy.fixture('buckets').then((buckets) => {
+    cy.get('[data-testid="bucket-form-name"]').type(
+      `${buckets.newBucket.name}-${random}`
+    );
+    cy.get('[data-testid="bucket-form-submit"]').click();
+    cy.get(`[data-testid="bucket-card ${buckets.newBucket.name}-${random}"]`);
+  });
+});
+
+it('allows to import and visualize new data to DB', () => {
+  const MAX_SAMPLE_VALUE = '2.43';
```

Comment:
Why is not in the cypress.json file? To me is weird having a parameter there hardcoded.